src/ansys/dpf/core/result_info.py

```diff
@@ -61,14 +65,25 @@ class ResultInfo:
 
     This class describes the metadata of the analysis and the available results.
 
+    .. note::
+      Creating a new ResultInfo from an analysis type and physics type is currently only available
```

Comment:
I see that you add more more capabilities available witha single server type, I'm afraid that it's going to make the library hard to use :/ (I found the issue with the streams class 2 days ago)

Reply:
Are you following up to expand to other server types after?

Reply:
@cbellot000 definitely, yes. But this can be merged first to allow for a custom operator to create a ResultInfo.